packages/reports/addon/components/filter-values/multi-value-input.ts

```diff
@@ -31,12 +39,12 @@ export default class MultiValueInput extends Component {
    * @param {String} tag - Add single value to the filter values list
```

Comment:
Remove extra type info from docs
```suggestion
   * @param tag - Add single value to the filter values list
```